### DISCUSSION TITLE
New definitions of pre-extensively and extensively drug-resistant tuberculosis

### DISCUSSION THREAD

**Discussion by dmujuni** [jodyphelan/TBProfiler#175] [Ideas]
Hi, I would like to know if the latest version of TBProfiler classifies drug resistance of isolates according to the New definitions of pre-extensively and extensively drug-resistant tuberculosis as per the update from the World Health Organization. Thanks for the tremendous work done on the pipeline.

**Comment by jodyphelan** [comment]
Hi @dmujuni 

It is actually still using the old definition. I'll get that updated in the next release!

Edit: This is now updated to the latest definitions

**Reply by dmujuni** [reply]
Alright, cool 👍
